src/server/routes/api/document/index.js

```diff
@@ -619,32 +620,8 @@ const generateSitemap = () => {
  *          application/zip:
  *             description: Download a zip file containing each Document in various file formats.
  */
-http.get('/zip', function( req, res, next ){
-  let filePath = 'download/factoid_bulk.zip';
-
-  const lazyExport = () => {
-    let recreate = true;
-    if ( fs.existsSync( filePath ) ) {
-      const DAY_TO_MS = 86400000;
-
-      let now = Date.now();
-      let fileDate = fs.statSync(filePath).birthtimeMs;
-
-      if ( now - fileDate < DAY_TO_MS ) {
-        recreate = false;
-      }
-    }
-
-    if ( recreate ) {
-      let addr = req.protocol + '://' + req.get('host');
-      return exportToZip(addr, filePath);
-    }
-
-    return Promise.resolve();
-  };
-
-  tryPromise( lazyExport )
-    .then( () => res.download( filePath ))
+http.get('/zip', async function( req, res, next ){
+  tryPromise( () => res.download( BULK_DOWNLOADS_PATH ) )
```

Comment:
It could be useful to log errors here in case the generation of the zip somehow failed